### PR TITLE
Docker: Dockerize Webpacker

### PR DESCRIPTION
**Pull request by gr455**
Follow up to #1779 

## This pull
- Creates a seperate container for Webpacker on default network

### CODE REVIEW

**Comment by gr455** [comment]
@tachyons what do you think about this

**Comment by tachyons** [comment]
https://github.com/rails/webpacker/blob/master/docs/docker.md this is the officially recommended setup

**Comment by gr455** [comment]
> https://github.com/rails/webpacker/blob/master/docs/docker.md this is the officially recommended setup

Thanks for sharing that, this will hopefully save a lot of trouble during installation

File: docker-compose.yml [hunk 1]
```diff
@@ -43,11 +44,12 @@ services:
       - NODE_ENV=development
       - RAILS_ENV=development
       - WEBPACKER_DEV_SERVER_HOST=0.0.0.0
-    command: ./bin/webpack-dev-server
+    entrypoint: /circuitverse/bin/docker_webpacker_run
     volumes:
       - .:/circuitverse
-    ports:
-      - '127.0.0.1:3035:3035'
```
Comment (gr455) [inline docker-compose.yml:50]: Do we need to expose the port?

Comment (tachyons) [reply docker-compose.yml:50]: This for assets, so better keep this

Comment (gr455) [reply docker-compose.yml:50]: @tachyons is this fine?

**Comment by satu0king** [comment]
Great work @gr455, its working fine on my laptop. I think the local dev setup can be further optimized, will create an issue with some ideas. Merging for now.

**Pull request merged**